c/indexer.c

```diff
@@ -590,7 +590,6 @@ struct gzip_index* blob_to_index(void* buf)
         cur += 8;
         memcpy(&pt->out, cur, 8);
         cur += 8;
-        memset(&pt->bits, 0, sizeof(int));
         memcpy(&pt->bits, cur, 1);
```

Comment:
Or the less obfuscated
```
pt->bits = *(uint8_t *)cur;
```
and similarly for `pt->in` and `pt->out`, above. But okay.

Reply:
Given these recommended changes are unrelated to the PR I'd prefer this to be done in a separate PR just incase it introduces something.  If we even want this.